### PR TITLE
Update robot.names

### PR DESCRIPTION
**Pull request by UltraHex**
Added robot names to round out existing sources I'm familiar with and fixed alphabetization.

### CODE REVIEW

File: src/main/resources/assets/opencomputers/robot.names [hunk 1]
```diff
@@ -83,7 +88,9 @@ Roomba             # Under your couch... wait.
 Rosie              # The Jetsons
 Shakey             # The first general-purpose mobile robot that could reason about its actions.
 Skynet             # Terminator
+Sonny              # I, Robot
 Space Core         # Portal
+Spirit              # Mars Rover
```
Comment (Vexatos) [inline src/main/resources/assets/opencomputers/robot.names]: ~~AAAARGH MY EYES!~~

Please fix the indentation here.

**Comment by fnuecke** [comment]
Yes please, nitpicking where it's due ;-)

After that, please squash.

**Comment by UltraHex** [comment]
Fixed it, but I'm not sure what you mean by "squash."

**Comment by fnuecke** [comment]
[How to squash](http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git/5201642#5201642).

**Comment by payonel** [comment]
never squashed. feel free to open a new PR if you come back to this.

**Pull request closed without merging**